com.ibm.wala.cast.python/source/com/ibm/wala/cast/python/ipa/callgraph/PythonSSAPropagationCallGraphBuilder.java

```diff
@@ -271,6 +290,33 @@ protected void processCallingConstraints(
     }
   }
 
+  private IClass getCallable(CGNode target) {
+    IMethod method = target.getMethod();
+    IClass declaringClass = method.getDeclaringClass();
+    TypeName declaringClassName = declaringClass.getName();
+
+    TypeReference typeReference =
+        TypeReference.findOrCreate(
+            declaringClass.getClassLoader().getReference(), declaringClassName);
+
+    return this.getClassHierarchy().lookupClass(typeReference);
+  }
```

Comment:
I don't understand what this method is doing.  To me it looks like it just returns `declaringClass`.  What am I missing?  Can we add some documentation?